cmd/fleetctl/testing_utils.go

```diff
@@ -15,6 +15,8 @@ import (
 	"github.com/urfave/cli/v2"
 )
 
+var DefaultPassword = "password123#"
```

Comment:
Do we need to export this variable?

Reply:
@lucasmrod, it is being used in `cmd/fleetctl/users_test` at the moment. I also thought we might want to have something that is more widely available so future changes to password requirements would be easier to implement compared to the changes that were needed for this PR. What do you think?

Reply:
>  it is being used in cmd/fleetctl/users_test at the moment. 

OK, if we leave it here, then I would leave it unexported for now:
1. it's clear for readers that the type is only used in the package.
2. static check tools can detect if a private variable/method is not being used (and therefore can be deleted).

> I also thought we might want to have something that is more widely available so future changes to password requirements would be easier to implement compared to the changes that were needed for this PR. What do you think?

We could move `var DefaultPassword = "password123#"` to the `test` package in `server/test`.

Reply:
Ah ok, I already have an exported variable in the `test` package that we can also use here :)